documentation/modules/proc-deploy-service-monitors.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// assembly-monitoring-kube.adoc
+// assembly-monitoring-oc.adoc
+
+[id='deploy-service-monitors-{context}']
+= Deploying Service Monitors
+
+Follows these steps to enable Prometheus to monitor {ProductName}.
```

Comment:
Change to: You must deploy the service monitors to enable Prometheus to monitor {ProductName}.